api/src/main/scala/hmda/api/http/public/HmdaFilerPaths.scala

```diff
@@ -54,6 +61,47 @@ trait HmdaFilerPaths extends HmdaCustomDirectives with HmdaFilerProtocol with Ap
             sendResponse(filersF, uri)
           }
         }
+      } ~
+      path("filers" / Segment / Segment / "msaMds") { (period, instId) =>
```

Comment:
Having this as a separate, named path would make this easier to read and maintain. e.g. `def hmdaFilerMsasPath = `